pageserver/src/repository.rs

```diff
@@ -33,12 +45,30 @@ pub trait Timeline {
     /// Does relation exist?
     fn get_relsize_exists(&self, tag: RelTag, lsn: Lsn) -> Result<bool>;
 
-    // Functions used by WAL receiver
+    //------------------------------------------------------------------------------
+    // Public PUT functions, to update the repository with new page versions.
+    //
+    // These are called by the WAL receiver to digest WAL records.
+    //------------------------------------------------------------------------------
```

Comment:
@ericseppanen, is there some Rust convention for "section divider" comments like this?

Reply:
Not that I know of.